sdk/translation/ai-translation-document-rest/test/public/node/documentTranslationTest.spec.ts

```diff
@@ -50,6 +51,7 @@ describe("DocumentTranslation tests", () => {
   let client: DocumentTranslationClient;
 
   beforeEach(async function (this: Context) {
+    setLogLevel("verbose");
```

Comment:
Please remember to remove this `setLogLevel` call now that we've finished debugging. It'll cause a _ton_ of noise in CI

Reply:
Removed